lib/dogapi/facade.rb

```diff
@@ -549,8 +549,8 @@ def can_delete_monitors(monitor_ids)
       @monitor_svc.can_delete_monitors(monitor_ids)
     end
 
-    def delete_monitor(monitor_id)
-      @monitor_svc.delete_monitor(monitor_id)
+    def delete_monitor(monitor_id, options = {})
```

Comment:
This means existing users that aren't passing in `options` will still be able to omit it, right?